data/human/human missions.txt

```diff
@@ -6301,3 +6301,410 @@ mission "Blind Man from Martini"
+			`	She is lying on a stretcher placed on what appears to be a folding cafeteria table, with an IV in one arm and a makeshift bandage over the other shoulder, already soaked through with blood. The lower half of her body is covered by a bloody sheet...and by the shape the sheet makes, you can tell immediately that there's something very wrong. Her eyelids flicker for a moment, then she opens her eyes and focuses on you. "Hey...I know you. You brought me that package from Tor...thanks. Meant a lot." She attempts to raise her head, then thinks better of it, though her eyes still flick downward. "Might be...the last of home I get to see." Her eyes close again, and a pair of nurses comes over and shoos you out of the way so they can start moving Irene. As they carry her off, careful to keep her IV bag held high, you hear her start coughing, and they head to what appears to be the largest of the ships nearby.`
+				decline
+
+mission "Take Me To The South A: Followup A1"
+	landing
+	invisible
+	to offer
+		has "Take Me To The South A: Followup A: declined"
+		has "FW Pirates: Attack 3: done"
+	on offer
+		conversation
+			`Now that you have a minute to breathe, you turn to your computer and pull up the casualty lists from the Battle of Greenrock. You only have to scroll through it for a few seconds before you see what you are looking for and dreading:`
+			`	`
+			`		MSGT BROWER, TORREY`
+			`			KILLED IN ACTION`
+			`			POSTHUMOUS MEDAL OF VALOR`
+			``
+			`	You kill the display and turn away.`
+				decline
+				
+mission "Take Me To The South A: Followup B"
+	landing
+	invisible
+	source "Deep"
+	to offer
+		has "delivered torrey's package to irene"
+		has "FW Pirates: Diplomacy 4: done"
+	on offer
+		conversation
+			`As you leave the quartermaster's building, you spot a familiar face coming in the other direction: Irene Brower, the gunnery sergeant you delivered a care package to before the war, though she seems to have master sergeant insignia now. She's on crutches, with what looks like a brand-new prosthetic leg. She spots you at about the same time, and her face, though lined with pain, brightens. She limps over. "Hey, I know you! You brought me that package from Tor, before - thanks!" She shifts her weight, and something evidently twinges, for she winces and shifts it back.`
+			`	You open your mouth, then shut it again, feeling a little awkward and unsure what to say about Irene's condition. She immediately notices your embarrassment, and grins lopsidedly. "Yeah, I was a casualty of the Battle of Greenrock. My CO thought it was a good idea to follow Councilor Voigt's plan to suppress the pirates there - to be honest, I agreed with him at the time." She pauses a moment. "Now, not so sure, and not just because a blown-out shield capcitor lost me a leg. I heard what happened on Thule, both the good and the bad of it. I'm not sure I can really say I think force was the best answer now."`
```

Comment:
```suggestion
			`	You open your mouth, then shut it again, feeling a little awkward and unsure what to say about Irene's condition. She immediately notices your embarrassment, and grins lopsidedly. "Yeah, I was a casualty of the Battle of Greenrock. My CO thought it was a good idea to follow Councilor Voigt's plan to suppress the pirates there - to be honest, I agreed with him at the time." She pauses a moment. "Now, not so sure, and not just because a blown-out shield capacitor lost me a leg. I heard what happened on Thule, both the good and the bad of it. I'm not sure I can really say I think force was the best answer now."`
```